src/Endpoints/README.md

```diff
@@ -0,0 +1 @@
+# wp-parsely Plugin WordPress REST API Endpoints
```

Comment:
We should pull out `src/Endpoints/README.md` to separate PR & flesh it out with information about how we build our endpoints.

Reply:
Now tracked in #702.